polkadot/node/service/src/chain_spec.rs

```diff
@@ -158,15 +159,17 @@ fn default_parachains_host_configuration(
 			allowed_ancestry_len: 2,
 		},
 		node_features: bitvec::vec::BitVec::from_element(
-			1u8 << (FeatureIndex::ElasticScalingMVP as usize),
+			1u8 << (FeatureIndex::ElasticScalingMVP as usize) |
+				1u8 << (FeatureIndex::EnableAssignmentsV2 as usize),
 		),
 		scheduler_params: SchedulerParams {
 			lookahead: 2,
 			group_rotation_frequency: 20,
 			paras_availability_period: 4,
 			..Default::default()
 		},
-		..Default::default()
+		approval_voting_params: ApprovalVotingParams { max_approval_coalesce_count: 5 }..
```

Comment:
@alexggh I found 5 to be a good arbitrary value to start with on testnets. For prod networks I'd g make it equal to `relay_vrf_module_samples` to optimize tranche0 signatures as much as possible.